docs/clients/cloud-watch-logs.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: client
+category: clients
+name: CloudWatchLogs
+package: async-aws/cloud-watch-logs
+---
+
+## Usage
+
+### List log streams
+
+```php
+use AsyncAws\CloudWatchLogs\CloudWatchLogsClient;
+use AsyncAws\CloudWatchLogs\Input\DescribeLogStreamsRequest;
+
+$cloudWatchLogs = new CloudWatchLogsClient();
+
+$result = $cloudWatchLogs->describeLogStreams(new DescribeLogStreamsRequest([
+    'logGroupName' => 'log-group-name',
+]));
+
+foreach ($result->getLogStreams() as $stream) {
+    echo '-'.$stream->getLogStreamName().': '.$stream->getSequenceToken().PHP_EOL;
```

Comment:
`getSequenceToken` does not exists in `Stream`. I think you mean `getUploadSequenceToken`

Reply:
but you can `echo '-'.$stream->getLogStreamName().PHP_EOL;` would also be fine

Reply:
The point was to document where the sequence token for the push logs call comes from

Too bad getLogStreams's return type is not recognized by PHPStorm, do you use a plugin for that?

Reply:
Because, it return anobject that implement IteratorAggregate and nothing says that type of yield items.

I'll create a PR to add a typehint on client's operation